### ISSUE TITLE
HLS starts at 10s

### ISSUE THREAD

**Issue by mseeley** [google/shaka-player#1793]
**Have you read the [FAQ](https://bit.ly/ShakaFAQ) and checked for duplicate open issues?**
Yes

**What version of Shaka Player are you using?**
- Shaka: 2.4.6, 2.4.5, and 2.3.3.
- Our custom app combines Shaka v2.3.3 + mux.js v5.0.1. We are trying to update to Shaka >= v2.4.6 + mux.js v5.0.1.

**Can you reproduce the issue with our latest release version?**
Yes

**Can you reproduce the issue with the latest code from `master`?**
No, I honestly have not tried.

**Are you using the demo app or your own custom app?**
Demo and custom app.

**If custom app, can you reproduce the issue using our demo app?**
Yes

**What browser and OS are you using?**
Chrome 72, macOS 10.14.1

**For embedded devices (smart TVs, etc.), what model and firmware version are you using?**
n/a

**What are the manifest and license server URIs?**

- https://provider-media.plex.tv/22fedd71260e2e27c7025a751f800d97/video.m3u8
- Manifest and segments can be [downloaded](https://drive.google.com/file/d/10nAMHoAyLpppI7eZJ2eTJO5y2szChWad/view?usp=sharing) separately for local testing

**What did you do?**

- Load the [2.4.6 demo directly](https://shaka-player-demo.appspot.com/demo/#asset=https://provider-media.plex.tv/22fedd71260e2e27c7025a751f800d97/video.m3u8;lang=en-US;build=uncompiled) to the manifest above
  - `Maximum small gap size:` should be default 0.5
  - `Jump large gaps` is unchecked

**What did you expect to happen?**

Playback should begin at 0s. In our app, v2.3.3 + mux.js v5.0.1 play the stream from start time 0 without `manifest.defaultPresentationDelay` nor `manifest.hls.defaultTimeOffset` values. I believe there is a behavior change between 2.3.3 and 2.4.5 that I haven't been able to identify.

**What actually happened?**

- Stream jumps forwards to ~10s.
- Log displays `Jumping forward 0.1154080000000004 seconds because of gap starting at 11.862333 and ending at 11.947666`

Our upgrade to 2.4.5 and 2.4.6 both demonstrate that stream playback now starts at ~10s. The delay smells like a pre-roll presentation delay of some sort. It did not occur in 2.3.3.

These issues describe a 10s skip at start: #1625, #1011. The combined configuration workaround (shown below) doesn't change behavior.

```
this._shaka.configure({
  manifest: {
    // #1625
    defaultPresentationDelay: 0,
    hls: {
      // #1011
      defaultTimeOffset: 0
    }
  },
  streaming: {
    bufferingGoal: this.props.bufferingGoal,
    jumpLargeGaps: true,
    retryParameters: {
      maxAttempts: Infinity
    }
  },
  abr: {
    defaultBandwidthEstimate: this.props.defaultBandwidthEstimate
  }
});
```

![kapture 2019-02-05 at 14 57 00](https://user-images.githubusercontent.com/30019/52314374-6a5e4500-2956-11e9-8f4f-b947721b978d.gif)

**Comment by vaage** [comment]
@mseeley Most HLS content starts at 10 seconds, this is something that Apple's HLS packager does and so a lot of content starts at 10 seconds.

Is the player:
 - skipping the first 10 seconds of your content

or is the player:
 - saying your contents at timer=10 seconds and no actual content was skipped

**Comment by mseeley** [comment]
HI @vaage good question. I've just confirmed; the player is skipping the first 10s of content.

**Comment by mseeley** [comment]
@vaage, #1625 mentions [a fix in 2.50 beta](https://github.com/google/shaka-player/pull/1602#issuecomment-437518884). I can confirm this is fixed in the 2.5.0-beta2 npm package.

**Comment by vaage** [comment]
@joeyparrish was #1602 cherry-picked into 2.4 (I don't see it in the changelog)? If not, is there any way to flag it to be cherry-picked in the future?

**Comment by joeyparrish** [comment]
It was not, but I just pushed it to v2.4.x in cd9d066386180028c8e0078ea7171163fae34775, which will be released in v2.4.7.  Sorry for the inconvenience!  I didn't realize we had missed that fix.

**Comment by mseeley** [comment]
Thank you @vaage and @joeyparrish! Zero worries here. 😄

**Comment by mseeley** [comment]
Hi @joeyparrish 👋. I've been sleuthing in this repo to find a general 2.4.7 release plan/timing although I haven't found information. Will you please share some general thoughts on 2.4.7? No pushy request to release, just a request for a bit of context I can use to update state on our internal issue tracker. Thanks!

**Comment by joeyparrish** [comment]
Hi, @mseeley.  I'm working on it today.  We're a bit behind on everything, which is hard to tell since I haven't published a timeline or roadmap yet.  That's coming soon, too, though.

**Comment by joeyparrish** [comment]
Just noticed that this is still in "needs triage".  I'm going to go ahead and close as a duplicate of #1602 based on the history so far.  The fix for that is in `master` (so you can try it out any time) and will be rolled into v2.4.7 very soon.  Thanks!

**Comment by joeyparrish** [comment]
Fix released in v2.4.7.
